Hangman.py

```diff
@@ -0,0 +1,68 @@
+# This is a word game
+from random import choice
+from collections import Counter
+
+
+def selectWord(fname):
+    words = open(fname).readlines()
+    myword = choice(words)
+    myword = str(myword).strip('  \n')
+    myword = myword.lower()
+    return myword
+
+
+def LetterFrequency(text,size):
+    counts = Counter(c for c in text if  c.strip("\n")).most_common(1)[0]
+    percentege = '{0:.2f}'.format( 100*(counts[1]/ float(size)))
+    print (f'{counts[0].lower()}  -  appears {counts[1]} ({percentege} %) times in the word.)')
+    return str(counts[0])
+
+
+def RemoveLetter(letter,text):
+    return text.replace(letter,"")
+
+
+def Main():
+  print("Let's play the game\n")
+  #select file to open
+  while True:
+      try:
+         fname = input('Enter the file name -> ')
+         break
+      except:
+         print('This file does not exists. Please try again!')
+         continue

```

Comment:
This code block will never execute. 
You can learn more here: [How to Check if a File or Directory Exists in Python](https://linuxize.com/post/python-check-if-file-exists/#check-if-file-exists-using-the-ospath-module)